components/EmbedMedia/EmbedMedia.styles.ts

```diff
@@ -1,4 +1,4 @@
 export const mediaWrapper = '*:!w-full *:!h-full';
-export const caption = '*:*:leading-display caption mt-1em max-w-prose-wide';
```

Comment:
Remove redundant caption style from embed media caption because `figcaption` HTML element already has the font size built in